ciao-controller/controller_test.go

```diff
@@ -1122,7 +1122,7 @@ func TestGetStorageForVolume(t *testing.T) {
 		t.Errorf("bootable flag not correct")
 	}
 
-	if pl.Ephemeral != false {
+	if pl.Ephemeral != true {
```

Comment:
@rbradford why is this one automatically changing its ephemerality?  Is this because it's an autocreated overlay volume created from another volume?